test/algorithms/buffer/test_buffer.hpp

```diff
@@ -357,7 +358,8 @@ void test_buffer(std::string const& caseid,
 
     if (settings.test_area)
     {
-        auto const area = bg::area(buffered, strategy);
+        bg::strategies::buffer::geographic<bg::strategy::karney, bg::srs::spheroid<double>> kstrategy;
```

Comment:
Is `test_buffer` function only used for geographic CS?

Since different spheroids are used above, shouldn't they be passed here, i.e. extracted from `strategy`? This probably is not a problem since the issue is not correctness but consitency across compilers but still.